repos/rust-lang/cc-rs.toml

```diff
@@ -0,0 +1,10 @@
+org = 'rust-lang'
+name = 'cc-rs'
+description = 'Rust library for build scripts to compile C/C++ code into a Rust library'
+bots = []
+
+[access.teams]
+crate-maintainers = 'write'
```

Comment:
Is there a reason this one is `write` and the others are `maintain` (caveat: I don't know what the difference is between these).

Reply:
Nope - just a typo.